storybook/stories/components/search/filters.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import React from 'react'
+import { Col, Grid } from 'reflex'
+import styles from '../../../shared/stories.css'
+
+export default function SearchFilters() {
+  return (
+    <Col size={3} className={styles.colors4} align="top">
```

Comment:
I agree, and had considered it. I didn't go for it since I thought it would make the independent story look awkward.

But having a flexible, independent story seems more compelling and useful